common/models/user.js

```diff
@@ -475,13 +602,13 @@ module.exports = function(User) {
    */
 
   User.prototype.verify = function(options, fn) {
+    options = options || this.getVerifyOptions();
```

Comment:
implementing options defaulting in prototype method as well.

if fine with you i will rename options as verifyOptions to make sure there's no confusion with REST options used in User.verify()